zeroforcing/metagraph.pyx

```diff
@@ -32,6 +32,17 @@ from cpython.mem cimport (
 from zeroforcing.fastqueue cimport FastQueueForBFS
 
 
+def zero_forcing_set(sage_graph):
+    cdef:
+        ZFSearchMetagraph metagraph = ZFSearchMetagraph(sage_graph)
+        frozenset start = frozenset()
+        frozenset end = frozenset(metagraph.to_relabeled_metavertex(sage_graph.vertices(sort=False)))
+
+    return metagraph.dijkstra(start, end)
+
+def zero_forcing_number(sage_graph):
+    return len(zero_forcing_set(sage_graph))
+
```

Comment:
Do you think this function is even necessary? IMO, it should be pretty self-explanatory for them to get the zero forcing number if we just give them `zero_forcing_set()`.